website/docs/r/data_factory_dataset_parquet.html.markdown

```diff
@@ -0,0 +1,135 @@
+  location = "West Europe"
+}
+
+resource "azurerm_data_factory" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_data_factory_linked_service_web" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  data_factory_name   = azurerm_data_factory.example.name
+  authentication_type = "Anonymous"
+  url                 = "https://www.bing.com"
+}
+
+resource "azurerm_data_factory_dataset_parquet" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  data_factory_name   = azurerm_data_factory.example.name
+  linked_service_name = azurerm_data_factory_linked_service_web.example.name
+
+  http_server_location {
+    relative_url = "http://www.bing.com"
+    path         = "foo/bar/"
+    filename     = "fizz.txt"
+  }
+}
+```
+
+## Argument Reference
+
+The following supported arguments are common across all Azure Data Factory Datasets:
+
+* `name` - (Required) Specifies the name of the Data Factory Dataset. Changing this forces a new resource to be created. Must be globally unique. See the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/data-factory/naming-rules) for all restrictions.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Data Factory Dataset. Changing this forces a new resource
+
+* `data_factory_name` - (Required) The Data Factory name in which to associate the Dataset with. Changing this forces a new resource.
+
+* `linked_service_name` - (Required) The Data Factory Linked Service name in which to associate the Dataset with.
+
+* `folder` - (Optional) The folder that this Dataset is in. If not specified, the Dataset will appear at the root level.
+
+* `schema_column` - (Optional) A `schema_column` block as defined below.
+
+* `description` - (Optional) The description for the Data Factory Dataset.
+
+* `annotations` - (Optional) List of tags that can be used for describing the Data Factory Dataset.
+
+* `parameters` - (Optional) A map of parameters to associate with the Data Factory Dataset.
+
+* `additional_properties` - (Optional) A map of additional properties to associate with the Data Factory Dataset.
+
+The following supported locations for a Parquet Dataset:
+
+* `http_server_location` - (Required) A `http_server_location` block as defined below.
+
+* `azure_blob_storage_location` - (Required) A `azure_blob_storage_location` block as defined below.
+
+The following supported arguments are specific to Parquet Dataset:
+
+* `compression_codec` - (Optional) The compression codec used to read/write text files. Valid values are `bzip2`, `gzip`, `deflate`, `ZipDeflate`, `TarGzip`, `Tar`, `snappy`, or `lz4`. Please note these values are case sensitive.
+
+---
+
+A `schema_column` block supports the following:
```

Comment:
does schema make sense here?
```suggestion
A `column` block supports the following:
```
or
```suggestion
A `column_schema` block supports the following:
```

Reply:
As discussed, this is a shared property among all datasets, renaming it will introduce inconsistency.